data/worlds/chaosdorf/areas/floor.toml

```diff
@@ -0,0 +1,64 @@
+name = "floor"
+description = "a long floor with various doors, a wardrobe, and some dimmed lights"
+
+[[contents]]
+kind = "gateway"
+name = "hackcenter door"
+target = "hackcenter"
+obvious = true
+
+[[contents]]
+kind = "gateway"
+name = "everyones toilet door"
+target = "everyones_toilet"
```

Comment:
This should be `-`, right?

Reply:
we could add this as a requirement and write a check for this? Then it would be like the `-` in translation placeholders and area filenames.